deployment-examples/aws/k8s/deploy_to_eks.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+set -e
+
+## edit those environment variables
+## ----------------------
+export IMAGE_BASE_NAME=$AWS_ECR_REPOSITORY
+export TAG="0.0.1"
+export REGION_CODE=$AWS_REGION_CODE
+export CLUSTER_NAME=$AWS_CLUSTER_NAME
+## ----------------------
+
+export HELM_HOME="$HOME/helm"
+export HELM_VERSION="v3.5.2"
+export HELM_VALUES_DIR="./k8s"
+export HELM_CHARTS_REPO="https://github.com/commercetools/k8s-charts.git"
+export HELM_CHARTS_VERSION="1.14.0"
+export HELM_CHART_TEMPLATE_NAME="cronjob"
+export HELM_CHARTS_LOCAL_FOLDER="charts-templates"
+
+
+printf "\n- Build and push docker images to AWS ECR\n"
+ECR_PATH=$AWS_ECR_PATH
+IMAGE_FULL_NAME=${ECR_PATH}/${IMAGE_BASE_NAME}
+
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin $ECR_PATH
+docker build -t $IMAGE_BASE_NAME:$TAG .
+docker tag $IMAGE_BASE_NAME:$TAG $IMAGE_FULL_NAME:$TAG
+docker push $IMAGE_FULL_NAME:$TAG
```

Comment:
Can you push the docker image to docker hub instead of AWS? This is OS and we always publish the OS images to docker hub.

Reply:
Can you check if you have access to create new repository here? https://hub.docker.com/repositories

Reply:
No, I don't. It should be here: https://hub.docker.com/u/commercetools. Please ask ops or create support ticket for this.

Reply:
Operations team are also suggesting to use Container registry from cloud.
https://commercetools.slack.com/archives/C32RYRKRB/p1647859378534699

Reply:
OK, if it's open for everyone to download, then no problem. Just ensure it will be an open image.